tasks/deprecated/20230527-l2-gauge-checkpointer/readme.md

```diff
@@ -2,7 +2,7 @@
 
 > ⚠️ **DEPRECATED: do not use** ⚠️
 >
-> This version has been replaced for an updated version: [StakelessGaugeCheckpointer](../../20230731-stakeless-gauge-checkpointer/). The new version corrects the period used to perform the checkpoints to the start of the previous week, and adds the capability to checkpoint multiple gauges given their addresses in the same transaction.
+> This version has been replaced for an updated version: [StakelessGaugeCheckpointer](../20230731-stakeless-gauge-checkpointer/). The new version corrects the period used to perform the checkpoints to the start of the previous week, and adds the capability to checkpoint multiple gauges given their addresses in the same transaction.
 
 Deployment of the `L2GaugeCheckpointer` contract. It automates the process of performing checkpoints to stakeless root gauges.
```

Comment:
Do we want to also say "The latest version is ../../20230915-stakeless-gauge-checkpointer-v2"?

Reply:
I think it's fine as is; otherwise we'd need to update all previous versions every time we push an update.